conans/client/toolchain/base.py

```diff
@@ -13,7 +15,9 @@ def write_toolchain(conanfile, path, output):
                     conanfile.toolchain()
         else:
             try:
-                toolchain = {"cmake": CMakeToolchain}[conanfile.toolchain]
+                toolchain = {"cmake": CMakeToolchain,
+                             "make": MakeToolchain,
```

Comment:
Do not do unrelated things like "make" toolchain.

Reply:
okay, removed make (however, it seems like it was forgotten here - so maybe needs its own PR to fix)

Reply:
Yes, lets do it in another PR, anyway, we are doing lots of changes there.